addons/models/3000-Cloud/3030-aws_s3_v2_typedefs.json

```diff
@@ -251,6 +251,15 @@
           "isIndexable": false,
           "isOptional":  true,
           "isUnique":    false
+        },
+        {
+          "name":        "accountId",
+          "typeName":    "string",
+          "cardinality": "SINGLE",
+          "isIndexable": true,
+          "isOptional":  true,
+          "isUnique":    false,
+          "searchWeight" : 5
```

Comment:
if you want this attribute to be searchable in quicksearch/suggestions. Add a higher value for search weight. 9 or 10

Reply:
@sarathkumarsubramanian thanks for the suggestion.
done.